components/03-modules/slider/_slider.scss

```diff
@@ -1,15 +1,20 @@
-$slider__margin                    : $spacer--medium !default;
-$slider__transition                : $transition-base !default;
-$slider__item-width                : 50% !default;
-$slider__item-width--medium        : 33% !default;
-$slider__item-width--large         : 25% !default;
-$slider__navigation-margin         : $spacer !default;
-$slider__navigation-icon-size      : 48px !default;
-$slider__navigation-icon-margin    : $spacer !default;
-$slider__navigation-icon-background: $white !default;
-$slider__icon-size                 : 25px !default;
-$slider__icon-fill                 : $color-primary !default;
-$slider__icon-fill-hover           : darken($color-primary, 10%) !default;
+$slider__margin                       : $spacer--medium !default;
+$slider__transition                   : $transition-base !default;
+$slider__item-width                   : 50% !default;
+$slider__item-width\@medium           : 25% !default;
+$slider__navigation-margin            : $spacer !default;
+$slider__navigation-icon-size         : 48px !default;
+$slider__navigation-icon-margin       : $spacer !default;
+$slider__navigation-icon-background   : $white !default;
+$slider__icon-size                    : 25px !default;
+$slider__icon-fill                    : $color-primary !default;
+$slider__icon-fill-hover              : darken($color-primary, 10%) !default;
+$slider__dot-margin                   : $spacer--medium !default;
+$slider__dot-padding                  : $spacer--medium !default;
+$slider__dot-inside-size              : $spacer--medium !default;
+$slider__dot-inside-background        : $gray-dark !default;
+$slider__dot-inside-border-radius     : 8px !default;
+$slider__dot-inside-background\@active: $color-primary !default;
```

Comment:
it is not break point.  Should we use here `@` not regular modifier?

Reply:
replaced with regular modifier